util/cops/deprecations.rb

```diff
@@ -8,36 +8,24 @@ module Rubygems
       #
       # @example
       #
-      #   As of March, 2019
+      #   As of March, 2020
       #
       #   # bad
-      #   deprecate :safdfa, nil, 2018, 12
-      #   deprecate :safdfa, nil, 2019, 03
+      #   deprecate :safdfa, :none
       #
       #   # good
-      #   deprecate :safdfa, nil, 2019, 04
+      #   deprecate :safdfa
```

Comment:
In this case, the "good" alternative would be that the code is removed, but leaving an empty line could be confusing so maybe add a comment like:

```suggestion
      #   the `deprecate` call is fully removed
```